Detectors/FIT/common/simulation/src/Digitizer.cxx

```diff
@@ -46,8 +48,9 @@ void Digitizer::process(const std::vector<o2::fit::HitType>* hits, Digit* digit)
     for (int i = 0; i < parameters.mMCPs; ++i)
       channel_data.emplace_back(ChannelData{ i, 0, 0, 0 });
   }
+  Int_t parent = -10;
   assert(digit->getChDgData().size() == parameters.mMCPs);
-  for (auto& hit : *hits) {
+  for (auto& hit : sorted_hits) {
     Int_t hit_ch = hit.GetDetectorID();
```

Comment:
So, do you account in the energy loss the contribution from the "charged hits"?

Reply:
I do not collect energy loss, only number of photelectrons

Reply:
On line [66](https://github.com/AllaMaevskaya/AliceO2/blob/fd83b9684961b7838ebe843c4ff080b150abf959/Detectors/FIT/common/simulation/src/Digitizer.cxx#L66) you accumulate the registered e.loss of the hit as QTCampl, and the charged particles are registered with e.loss 10

Reply:
yes, and in my analysis i know number of charge particles as QTCaml/10 and number of photons as   number of particles -   QTCaml/10.
But you are right - i will fix this tricky massage.

Thank you
Alla